components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -417,7 +429,8 @@ RewardsServiceImpl::RewardsServiceImpl(Profile* profile)
       private_observer_(
           std::make_unique<ExtensionRewardsServiceObserver>(profile_)),
 #endif
-      next_timer_id_(0) {
+      next_timer_id_(0),
+      reset_states_(false) {
```

Comment:
can we please rename so the intention is clear, i.e. `is_resetting_state_`

Reply:
If we are going that way, I would like to see a document that describes all that requirements for the code. Put it in the repo, put it in the wiki.

Reply:
I will change when refactoring, thanks